src/app/model.ts

```diff
@@ -304,6 +304,15 @@ export interface OAuthApp extends BaseEntity {
 };
 export type OAuthApps = Array < OAuthApp > ;
 
+export interface Setup {
+  gitHubOAuthConfiguration: GitHubOAuthConfiguration;
+}
+
+export interface GitHubOAuthConfiguration {
+  clientId: string;
+  clientSecret: string;
+}
+
```

Comment:
If we keep adding to this file we may as well delete the generator code, as this is a generated file.

Reply:
Maybe we should create a `client-models.ts` file or something for this kind of thing...

Reply:
No opinion on this--it's up to you, just let me know what you prefer.

Reply:
Given that the swagger doc is broken anyways at the moment let's not worry about this.  Can you maybe just delete the `generate` build task from the package.json so we can't accidentally overwrite this file?